### DISCUSSION TITLE
Splash screen - image is perfectly sized on Android, about 5 pixels across on iOS. Helllllp!

### DISCUSSION THREAD

**Discussion by rogersm-uwosh** [dotnet/maui#19874] [Q&A]
I am *almost* ready to submit an app, but I cannot get the splash screen image to display properly on iOS. It looks (of course) fine on Android. 

The XAML is (I'm going to encode angle brackets just to be safe):

&lt;MauiSplashScreen Include="Resources\Images\biplane.svg" Color="#512BD4" BaseSize="512,512" /&gt;

I am attaching the image in question -- it's 512x512. On Android it looks fine, on iOS it is about 5 pixels across. Needless to say, this is not mentioned in the docs (AFAICT).

Honestly, I'm not sure I even need a splash screen (Apple discourages it unless really necessary), but deleting that line doesn't stop it from showing up. Any help in that area would also be appreciated.

![biplane](https://github.com/dotnet/maui/assets/91431120/d597c5b9-4405-4e57-af7d-80fa1c36978a)

**Comment by haavamoa** [comment]
We've added 2x svgs (`appiconios.svg` and  `appicon.svg`) for the different platforms, where the one on iOS has been slightly adjusted to make it perfect :

After the modifications, this can be added to your csproj:

```
    <!-- iOS app icon -->
        <MauiIcon Include="Resources\AppIcon\appiconios.svg" Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">
            <Color>#FFFFFF</Color> <!-- Background color -->
        </MauiIcon>

        <!-- Android app icon, weird viewbox due to an Resizetizer bug -->
        <MauiIcon Include="Resources\AppIcon\appicon.svg" Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">
            <Color>#FFFFFF</Color> <!-- Background color -->
        </MauiIcon>
```

**Reply by rogersm-uwosh** [reply]
That could be useful, although I still don't know why my icons are the wrong size. If I open them up in a browser or Omnigraffle they look ... big, on the app, miniscule. I'll start Googling or ChatGPTing Resizetizer.
